scripts/setup

```diff
@@ -370,8 +370,10 @@ then
     eval ${PREFIX} invenio oaiharvester harvest -n ebooks -q -k
 else
     eval ${PREFIX} invenio scheduler enable_tasks -n bulk-indexer -n notification-creation -n accounts -n clear_and_renew_subscriptions -v
-    info_msg "Start OAI harvesting 100"
-    eval ${PREFIX} invenio oaiharvester harvest -n ebooks -a max=100 -q
+    info_msg "For ebooks harvesting run:"
+    msg "\tinvenio oaiharvester harvest -n ebooks -a max=100 -q"
+    # info_msg "Start OAI harvesting 100"
+    # eval ${PREFIX} invenio oaiharvester harvest -n ebooks -a max=100 -q
```

Comment:
I suggest:

1. do not make ebook harvesting by default
2. add a parameter that activate the ebook harvesting. In deployment or not.